titanic.ipynb

```diff
@@ -43,9 +43,6 @@
         "# Load the Titanic dataset\n",
```

Comment:
_Commented on notebook `titanic.ipynb` Cell 2 From line 5 to 6_
```python
# Load the Titanic dataset
titanic = sns.load_dataset('titanic')
```
multi lin ehere

_Respond and view the context [here](http://localhost:3000/smith-kyle/notebook-diff-tests/pull/26)._